quantum/action_tapping.c

```diff
@@ -370,10 +374,15 @@ bool process_tapping(keyrecord_t *keyp) {
             if (event.pressed) {
                 if (IS_TAPPING_RECORD(keyp)) {
 //#    ifndef TAPPING_FORCE_HOLD
-#    if !defined(TAPPING_FORCE_HOLD) || defined(TAPPING_FORCE_HOLD_PER_KEY)
+#    if !defined(TAPPING_FORCE_HOLD) || defined(TAPPING_FORCE_HOLD_PER_KEY) || defined(TAPPING_FORCE_HOLD_TERM)
                     if (
-#        ifdef TAPPING_FORCE_HOLD_PER_KEY
+#        if defined(TAPPING_FORCE_HOLD_PER_KEY) && defined(TAPPING_FORCE_HOLD_TERM)
+                        // Apply force hold term only to per key
+                        (!get_tapping_force_hold(tapping_keycode, keyp) || WITHIN_FORCE_HOLD_TERM(event)) &&
+#        elif defined(TAPPING_FORCE_HOLD_PER_KEY) && !defined(TAPPING_FORCE_HOLD_TERM)
                         !get_tapping_force_hold(tapping_keycode, keyp) &&
+#        elif !defined(TAPPING_FORCE_HOLD_PER_KEY) && defined(TAPPING_FORCE_HOLD_TERM)
+                        WITHIN_FORCE_HOLD_TERM(event) &&
```

Comment:
```suggestion
                        (!get_tapping_force_hold(tapping_keycode, keyp) || !WITHIN_FORCE_HOLD_TERM(event)) &&
#        elif defined(TAPPING_FORCE_HOLD_PER_KEY) && !defined(TAPPING_FORCE_HOLD_TERM)
                        !get_tapping_force_hold(tapping_keycode, keyp) &&
#        elif !defined(TAPPING_FORCE_HOLD_PER_KEY) && defined(TAPPING_FORCE_HOLD_TERM)
                        !WITHIN_FORCE_HOLD_TERM(event) &&
```

Reply:
> The code looks good to me but in my opinion, the logic should be reversed.

I agree that your logic solves a different use-case. This PR solves a personal use-case where _slow_ same finger bigram on mod-tap keys are prevented from auto-repeating unless they are deliberately double-tap-held.

That said, logic for `TAPPING_FORCE_HOLD_TERM` can be reversed by introducing another flag/definition (in a subsequent PR) to swap that `if` condition.